Cargo.toml

```diff
@@ -7,14 +7,15 @@ license = "Apache-2.0/MIT"
 name = "addr2line"
 readme = "./README.md"
 repository = "https://github.com/gimli-rs/addr2line"
-version = "0.1.0"
+version = "0.2.0"
 
 [dependencies]
 fallible-iterator = "0.1.3"
-getopts = "0.2.14"
+clap = "2.*"
```

Comment:
I'd prefer specifying a proper version here. Cargo will automatically treat `2.3.6` as `^2.3.6`, which is anything in the range `[2.3.6, 3.0.0)`.